vault/resource_quota_rate_limit.go

```diff
@@ -43,6 +43,19 @@ func quotaRateLimitResource() *schema.Resource {
 				Description:  "The maximum number of requests at any given second to be allowed by the quota rule. The rate must be positive.",
 				ValidateFunc: validation.FloatAtLeast(0.0),
 			},
+			"interval": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Description:  "The duration in seconds to enforce rate limiting for (default of 1).",
+				ValidateFunc: validation.IntAtLeast(0),
+				Default:      1,
```

Comment:
Probably best to make this a `Computed` field, rather than assuming the default value from vault.

```suggestion
				Computed:      true,
```